Unix/buildtool

```diff
@@ -1753,6 +1755,108 @@ fi
 
 ##==============================================================================
 ##
+## 'ntlm' command 
+##     Whether gss negotiated ntlm is supported (1) or not (0)
+##
+##==============================================================================
+
+if [ "$arg1" = "ntlm" ]; then
+
+    if [ "$argc" != "1" -o "$opts" != "" ]; then
+        echo "Usage: $0 $arg1"
+        echo
+        exit 1
+    fi
+
+    case "$os" in
+        LINUX*)
+            # supported!
+            case "$distro" in
+                RedHat*)
+                    
+                    distro_major_Version=`echo $distro_version | awk -F'.' '{ print $1}'`
```

Comment:
Looks like `distro_major_Version` and `distro_minor_Version` can be calculated once outside the switch case since it's the same for all platforms.